_posts/2023-01-30-AddisonAdcock-the-process-meme.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Simple vs. Modern Operating Systems
+author: 
```

Comment:
I think I lost a comment somewhere...

The author should be Addison's short_name. Fix that and I can merge.

Reply:
I just committed and added my name. I think that was what was missing?

Reply:
yep! Thanks.